Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -568,6 +732,24 @@ import BraintreeDataCollector
         )
         completion(nil, BTPayPalError.canceled)
     }
+
+    private func notifyEditFISuccess(
+        with result: BTPayPalVaultEditResult,
+        completion: @escaping (BTPayPalVaultEditResult?, Error?) -> Void
+    ) {
+       // TODO: add Analytics Event
```

Comment:
When we do the analytics piece, lets make sure we are also passing the `correlationID` as the result of the data collector for success, failure, and cancel.